support/Pipelines/Bbh/InitialData.py

```diff
@@ -209,6 +209,8 @@ def generate_id(
       evolve: Set to True to evolve the initial data after generation.
       eccentricity_control: If set to True, an eccentricity reduction script is
         run on the initial data to correct the initial orbital parameters.
+      negative_expansion_bc: If set to True, the excision boundaries are set to
+        be inside of the apparent horizons.
```

Comment:
Please add something like "this helps to find horizons and start an evolution from the initial data without extrapolation. (default: True)"
Also add this to the CLI help below.